src/components/common/ColonyActions/ActionDetailsPage/ActionDetailsPageFeed/ActionDetailsPageFeed.tsx

```diff
@@ -10,21 +14,23 @@ const displayName =
 
 interface ActionsPageFeedProps {
   actionData: ColonyAction;
-  // networkEvents: FormattedEvent[];
 }
 
-const ActionDetailsPageFeed = ({
-  actionData,
-}: // networkEvents,
-ActionsPageFeedProps) => {
-  const events = ACTIONS_EVENTS[actionData.type];
+const ActionDetailsPageFeed = ({ actionData }: ActionsPageFeedProps) => {
+  const messages = actionData.isMotion
+    ? actionData.motionData?.messages
+    : ACTIONS_EVENTS[actionData.type];
   return (
     <>
-      {events?.map((event) => (
+      {messages?.map((message) => (
         <ActionsPageEvent
           actionData={actionData}
-          eventName={event}
-          key={event}
+          motionMessageData={message}
```

Comment:
This prop doesn't need to be present for actions, which is one of the reasons this doesn't feel like an appriopriate abstraction for the motions use case.